packages/gcm/lib/apk/config.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  - name: gcm::GcmService
+    registration-class: org.domokit.gcm.RegistrationIntentService$MojoService
+jars:
+  - $androidSdk/extras/google/google_play_services/libproject/google-play-services_lib/libs/google-play-services.jar
+  - $androidSdk/extras/android/support/v13/android-support-v13.jar
+  - $androidSdk/extras/android/support/v7/appcompat/libs/android-support-v7-appcompat.jar
+  - $androidSdk/extras/android/support/v7/mediarouter/libs/android-support-v7-mediarouter.jar
```

Comment:
E.g., this would be:

`- android-sdk:extras/android/support/v7/mediarouter/libs/android-support-v7-mediarouter.jar`